### ISSUE TITLE
Heading structure nested illogically in comment title

### ISSUE THREAD

**Issue by EvanHerman** [godaddy-wordpress/go#221]
**Describe the bug:**
The heading structure is not logically nested. This h3 element should be an h2 to be properly nested.

**Error Code:** WCAG2A.Principle1.Guideline1_3.1_3_1_A.G141

```bash
{ code: 'WCAG2A.Principle1.Guideline1_3.1_3_1_A.G141',
       context:
        '<h3 id="reply-title" class="comment-reply-title">Leave a Reply <small><a rel="no...</h3>',
       message:
        'The heading structure is not logically nested. This h3 element should be an h2 to be properly nested.',
       type: 'warning',
       typeCode: 2,
       selector: '#reply-title' }
```

**Screenshot(s):**
<img width="900" alt="Screen Shot 2019-08-14 at 4 46 08 PM" src="https://user-images.githubusercontent.com/5321364/63055053-0abb9600-beb3-11e9-84e7-11d09de61046.png">

<img width="1112" alt="Screen Shot 2019-08-14 at 4 47 58 PM" src="https://user-images.githubusercontent.com/5321364/63055161-4c4c4100-beb3-11e9-85cb-1feafc41ac9b.png">

**Priority** (select one):
- [x] High - Functional blocker
